Tests/FuzzilliTests/ProgramBuilderTest.swift

```diff
@@ -184,6 +184,81 @@ class ProgramBuilderTests: XCTestCase {
         let float3 = b.reuseOrLoadFloat(4.2)
         XCTAssertNotEqual(floatOutOfScope!, float3)     // Variable went out of scope
     }
+
+    // Open 2-3 scopes, have a single variable in the innermost scope (the outer ones are empty),
```

Comment:
Here and below, did you mean to leave these comments in here?
I guess they can stay here, but can be less vague (e.g. "Open 3 scopes", drop the "maybe ... if that's easy to do", etc.)

Reply:
ah no, these were mainly for my reference. I'll get rid of them now